src/commands/generate/index.ts

```diff
@@ -72,14 +73,20 @@ export default class Hello extends Command {
             },
           ],
         },
+        {
+          type: 'input',
+          name: 'awsRegion',
+          message: 'Which AWS Region do you choose? ',
```

Comment:
Should we add the default value `ap-southeast-1` as we often use that region?

Reply:
Added default region here: https://github.com/nimblehq/infrastructure-templates/pull/35/commits/20c518befd7c03b64fefb53bf438fe79bebd8374